src/queues.c

```diff
@@ -506,6 +506,27 @@ gint64 queues_get_next_datachange(gint64 time)
         return sleep != G_MAXINT64 ? sleep : -1;
 }
 
+
+
+
+/* see queues.h */
+struct notification* queues_get_received_by_id(int id)
```

Comment:
What does `received` mean in this context?
I'd say `queues_get_by_id` would be a shorter and more descriptive name.

Reply:
Hmmm I was trying to say that `id` was not looked for in all the queues… 
`queues_get_by_id`  would be a bit misleading as `history` queues is not used.

The best would probably to name the function `queues_get_by_id`   and search in all queues. In our use case it won't change the looking time as we expect the `id` to be in one of the first two queues.
Would this be ok ?

Reply:
That's fine by me. Default history max is ~20 items so performance impact is negligible.